client/src/components/related/relatedSpec.test.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+// Installation is suggest from React website
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
```suggestion

```
Linter may be throwing an error as a result of a bug. This normally happens after you install a package. ESLint may not recognize that you installed the package until you have restarted your VSCode. Please see me if it continues to throw error after doing the following:

1. delete line 3
2. make sure @testing-library/react is inside `package.json`
3. restart your vscode